lib/src/genius.dart

```diff
@@ -235,8 +255,8 @@ class Genius {
   }
 
   /// Uses beautiful_soup to scrape song lyrics off of a Genius song URL
-  static Future<String?> lyrics({required String url}) async {
-    String responseBody = (await http.get(Uri.parse(Uri.encodeFull(url)))).body;
+  Future<String?> lyrics({required String url}) async {
```

Comment:
![image](https://github.com/hbtalha/genius_lyrics/assets/32166914/49b1732b-f5f7-430a-b52a-deb235571d54)
why? if we put it static we have to do requestBody static too , and there's no need to do that , because we already create an instance of httclient

Reply:
@hbtalha here

Reply:
This function has to be static, should revert it

Reply:
![image](https://github.com/hbtalha/genius_lyrics/assets/32166914/c2971fd4-8ae8-44aa-bc2f-86622e33c074)
back to client , he's supost to take care of all api request , to make this method static we have to create our own parseUrl because we can't use non static methods inside statics 

this break the DRY ( Don't Repeat YourSelf ) principle , just to make this method static ?

Reply:
It is not always that you are able to follow the DRY strictly.
In this case, it is worth noting that the `requestBody` function as of right now is only used in this function.

I am aware of the implications of keeping this function static and it is stated in the previous comments the reasons why this function should remain static.

Reply:
i made a change and bonus, i follow the DRY :') , and the method will be static